### PR TITLE
Properly specify the round-trip format specifiers for floats and doubles in IMethodXML (used by WinForms)

### PR DESCRIPTION
**Pull request by DustinCampbell**
Fixes issue #4149.

Reviewers: @Pilchie, @jasonmalinowski

### CODE REVIEW

File: src/VisualStudio/Core/Impl/CodeModel/MethodXml/AbstractMethodXmlBuilder.cs [hunk 1]
```diff
@@ -464,8 +463,19 @@ protected void GenerateNumber(object value, ITypeSymbol type)
         {
             using (NumberTag(GetTypeName(type)))
             {
-                // TODO(DustinCa): Add more unit tests to ensure that floats are correct.
```
Comment (jasonmalinowski) [inline src/VisualStudio/Core/Impl/CodeModel/MethodXml/AbstractMethodXmlBuilder.cs]: :smile:

**Comment by jasonmalinowski** [comment]
:+1:

**Comment by DustinCampbell** [comment]
jenkins, retest this please

**Comment by jaredpar** [comment]
:+1:

**Pull request merged**